tensorflow/core/kernels/betainc_op.cc

```diff
@@ -33,6 +33,9 @@ namespace tensorflow {
 
 typedef Eigen::ThreadPoolDevice CPUDevice;
 typedef Eigen::GpuDevice GPUDevice;
+#ifdef TENSORFLOW_USE_SYCL
+typedef Eigen::SyclDevice SYCLDevice;
```

Comment:
using SYCLDevice = Eigen::SyclDevice

Reply:
Sometimes typedef is used, sometimes using is depending on the file. This on seem to use typedef. It's annoying but this is the kind of cleaning that should be done for the whole repository.